core/server/master/src/main/java/alluxio/master/lineage/LineageMasterClientServiceHandler.java

```diff
@@ -61,7 +61,7 @@
   /**
    * Creates a new instance of {@link LineageMasterClientServiceHandler}.
    *
-   * @param lineageMaster the {@link LineageMaster} the handler uses internally
+   * @param lineageMaster the {@link DefaultLineageMaster} the handler uses internally
```

Comment:
can you revert all the change from `LineageMaster` to `DefaultLineageMaster` in Javadoc like this, and below?